packages/desktop-gui/src/runs/dashboard-banner.jsx

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+
+export default class DashboardBanner extends Component {
```

Comment:
Can we make this a functional component instead of a class-based one? We have a lot of legacy class-based components of course, but for new ones, I think it's better to make them functional.